package.json

```diff
@@ -1,21 +1,21 @@
 {
-  "name": "ChronoFactorem",
+  "name": "chronofactorem",
```

Comment:
Why change the semantics of name?

Reply:
That was somehow changed by VSCode. Changed it back.